### PR TITLE
add hatch-docstring-description

### PR DESCRIPTION
**Pull request by flying-sheep**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/hatch-docstring-description/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - hatch_docstring_description
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/flying-sheep/hatch-docstring-description
+  summary: Hatch plugin to derive package description from Python package docstring
+  license: GPL-3.0
```
Comment (xhochy) [inline recipes/hatch-docstring-description/meta.yaml:38]: The linter is correct. This must either be `GPL-3.0-or-later` or `GPL-3.0-only`.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/hatch-docstring-description/meta.yaml (hunk 1 above)
Comment (flying-sheep) [reply recipes/hatch-docstring-description/meta.yaml]: well, that’s an actionable thing, since I’m the maintainer and can say what it meant.

**Comment by flying-sheep** [comment]
Hm, I did this PR for https://github.com/conda-forge/legacy-api-wrap-feedstock/pull/5, but it seems like that builds without having this build dependency installed? Is it enough to have build dependencies on PyPI?

**Comment by xhochy** [comment]
> Is it enough to have build dependencies on PyPI?

No, build dependencies also must be in conda-forge. Maybe in this case it worked because you are building from the sdist and the hatch plugin is only needed when generating the sdist and not during the actual installation process.

**Review by xhochy — APPROVED** [review]

**Comment by flying-sheep** [comment]
Ah, leaving out the build dep just happened to work. There’s not description in the installed metadata, but that’s not important.

**Pull request merged**